.env.test

```diff
@@ -35,6 +35,3 @@ OPENID_REFRESH_TOKEN="dummy-refresh-token"
 ACTIVE_RECORD_ENCRYPTION_PRIMARY_KEY="test_primary_key"
 ACTIVE_RECORD_ENCRYPTION_DETERMINISTIC_KEY="test_deterministic_key"
 ACTIVE_RECORD_ENCRYPTION_KEY_DERIVATION_SALT="test_derivation_salt"
-
-# Let the tests know if they are run within a Docker container:
-# DOCKER="true"
```

Comment:
Hehe, I would have left it in just as an example. It's easier to copy and paste from here then from the Readme. But it doesn't matter. The Readme has all the instructions to make this work.

Reply:
By the way, I think we could add this line in the `Dockerfile`. But I'm not able to test it out right now.